pysidt/plotting.py

```diff
@@ -1,27 +1,31 @@
 from IPython.display import Image, display
 import pydot
 import os
+import numpy as np
 
-
-def plot_tree(sidt, images=True):
+def plot_tree(sidt, images=True, depth=np.inf):
     graph = pydot.Dot("treestruct", graph_type="digraph", overlap="false")
     graph.set_fontname("sans")
     graph.set_fontsize("10")
     if not os.path.exists("./tree"):
         os.makedirs("./tree")
```

Comment:
Can we change all the use of `os` to `pathlib`? `pathlib` is the more modern way since Python 3.4
```suggestion
    graph.set_fontsize("10")
    save_dir = Path("./tree")
    save_dir.mkdir(exist_ok=True)
```